tests/mocha/workspace_test.js

```diff
@@ -120,19 +131,17 @@ suite('Workspace', function() {
       sinon.assert.calledOnce(stub);
       assertVariableValues(this.workspace, 'name1', 'type1', 'id1');
       assertVariableValues(this.workspace, 'name2', 'type2', 'id2');
-      var blockVarName1 = this.workspace.topBlocks_[0].getVarModels()[0].name;
-      chai.assert.equal(blockVarName1, 'name1');
-      var blockVarName2 = this.workspace.topBlocks_[1].getVarModels()[0].name;
-      chai.assert.equal(blockVarName2, 'name1');
```

Comment:
Change the name to be different so we can easily tell what test is failing.

Reply:
Added message to asserts in helper method and updated test names to help clarify what is being checked